analysis/src/python/evaluation/benchmark/benchmark.py

```diff
@@ -35,33 +35,61 @@ def from_value(cls, value: str) -> Optional['Analyzer']:
             return None
 
     @classmethod
-    def available_values(cls) -> Set[str]:
+    def values(cls) -> Set[str]:
         return {cls.HYPERSTYLE.value, cls.QODANA.value}
 
 
-def measure_run_time(f: Callable[[], Any], repeat: int, output_path: Path, parser: Callable[[Path], Any]) -> float:
+class AggregateOption(Enum):
```

Comment:
Can we unify this class with [this](https://github.com/hyperskill/hyperstyle-analyze/blob/main/analysis/src/python/data_analysis/template_errors/models/postprocessing_models.py#L10) class?